DigitalLearningSolutions.Web/Styles/shared/searchableElements/common.scss

```diff
@@ -1,5 +1,5 @@
-﻿@import "~nhsuk-frontend/packages/core/all";
-@import "../breakpoints";
+﻿@use "nhsuk-frontend/packages/core/all" as *;
+@use "../breakpoints";

```

Comment:
What does `as *` do? I noticed some files with "breakpoints" use  `as *`  and some don't.

Reply:
The `as` is a way to set the alias/namespace of the imported files, which is then used to specify the source of variables.

By default (sans `as`) the files have an alias of the file name. So, if we are using the `xl-desktop` variable from the breakpoints file in some styling, we would have to reference it as `breakpoints.$xl-desktop`. Setting the alias to `as *` means that we don't need to use the namespace to specify the variable, and we can just use `$xl-desktop`.

That change just reduces the amount of lines changed, and keeps the scss format the same as we had it before. If you have a file introduced with `@use` that does not have an alias, its contents are not directly references in the scss file it is being introduced to, but the page will still use its styles on the page's elements.

Doc page here: https://sass-lang.com/documentation/at-rules/use#choosing-a-namespace